fasta/faBin_test.go

```diff
@@ -78,8 +78,8 @@ func TestBinGenomeNoBreaks(t *testing.T) {
 	}
 
 	for i := range bins {
-		value, _ := bins[i]
-		minValue, _ := binsMin[i]
+		value := bins[i]
```

Comment:
It looks like in the old code bins[i] and binsMin[i] should each return 2 values. Do they only each return 1 value in the new code? Or is it ok now in golang to assign 2 values to 1 variable that accepts only the first value of the 2?